README.md

```diff
@@ -1 +1,162 @@
+    <li><a href="https://www.tvmaze.com/api">TVmaze API</a></li>  
+  </ul>
+</details>
+
+### Key Features <a name="key-features"></a>
+
+> Below are the key features of the website:
+
+- **Used TVmaze API for Show/Movies details and pictures**
+- **Used Involvement API for likes and comments interaction**
+- **Used webpack**
+- **Add counter tests with jest**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+- [Live Demo Link](https://yuvenalmash.github.io/API-based-webapp/dist/
+)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🚀 Video Representation <a name="video-rep"></a>
+
+- [Video Representation Link](https://drive.google.com/file/d/1OIBT3r2wCdjgWJ58tcUXIdAkCMSnopYg/view?usp=sharing)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up and running follow these simple example steps.
+
+1. Clone this repository or download the Zip folder:
+
+**``git clone https://github.com/yuvenalmash/API-based-webapp.git``**
+
+2. Navigate to the location of the folder in your machine:
+
+**``you@your-Pc-name:~$ cd <folder>``**
+
+### Visit And Open Files
+1-Visit Repo
+[Visit Repo](https://github.com/yuvenalmash/API-based-webapp)
+
+2-Download Repo
+[Download Repo](https://github.com/yuvenalmash/API-based-webapp/archive/refs/heads/main.zip)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 👥 Authors <a name="authors"></a>
+
+👤 **Yuvenal Njoroge**
+
+- GitHub: [@githubhandle](https://github.com/yuvenalmash)
+- Twitter: [@twitterhandle](https://twitter.com/_Juvenal_)
+- LinkedIn: [LinkedIn](https://linkedin.com/in/yuvenal-njoroge)
+
+👤 **Adeshina Adenuga**
+
+- GitHub: [@githubhandle](https://github.com/nuga0909)
+- Twitter: [@twitterhandle](https://twitter.com/nuga0909)
+- LinkedIn: [LinkedIn](linkedin.com/in/adeshina-adenuga-282036171)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🔭 Future Features <a name="future-features"></a>
+
+> Implement full schedule and search tab functionalities.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](https://github.com/yuvenalmash/API-based-webapp/issues).
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## ⭐️ Show your support <a name="support"></a>
+
+If you like this project, Give a ⭐️.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+- Project from [Microverse](https://www.microverse.org/?grsf=i6yi2m) html, css & Javascript module
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 📝 License <a name="license"></a>
+
+This project is [MIT](./MIT.md) licensed.
```

Comment:
Great job team 🤝

- [ ] [Optional] Kindly fix this broken MIT license link to make your readme file more professional.